src/SmartComponents/modules/reducers.js

```diff
@@ -145,7 +145,7 @@ export function compareReducer(state = initialState, action) {
                 totalFacts: filteredFacts.length
             };
         case `${types.EXPORT_TO_CSV}`:
-            reducerHelpers.downloadCSV(state.sortedFilteredFacts, state.systems);
+            reducerHelpers.downloadCSV(state.sortedFilteredFacts, [ ...state.systems, ...state.baselines ]);
```

Comment:
If we swap the order of these the CSV will match how its rendered in the UI (baselines first, then systems)